internal/gojs/compiler_test.go

```diff
@@ -89,7 +89,7 @@ func TestMain(m *testing.M) {
 
 	// Seed wazero's compilation cache to see any error up-front and to prevent
 	// one test from a cache-miss performance penalty.
-	rt = wazero.NewRuntimeWithConfig(testCtx, wazero.NewRuntimeConfig())
+	rt = wazero.NewRuntimeWithConfig(testCtx, wazero.NewRuntimeConfigInterpreter())
```

Comment:
here's where I temporarily switched to interpreter

Reply:
ic got it!